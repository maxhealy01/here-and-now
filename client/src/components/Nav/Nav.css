header {
	border-bottom: 1px var(--dark) solid;
	background-color: var(--tertiary);
}
li {
	list-style: none;
}
