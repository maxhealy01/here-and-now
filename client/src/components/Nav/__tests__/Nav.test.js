import React from "react";
import { render, cleanup, fireEvent } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Nav from "../Nav";
import { BrowserRouter as Router } from "react-router-dom";

afterEach(cleanup);

describe("Nav component", () => {
	it("renders", () => {
		render(
			<Router>
				<Nav />
			</Router>
		);
	});

	it("matches snapshot DOM node structure", () => {
		const { asFragment } = render(
			<Router>
				<Nav />
			</Router>
		);

		expect(asFragment()).toMatchSnapshot();
	});
});
