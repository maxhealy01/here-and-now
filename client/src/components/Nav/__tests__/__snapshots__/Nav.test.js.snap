// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <header
    class="flex-row px-1"
  >
    <h1>
      <a
        href="/"
      >
        Here and Now
      </a>
    </h1>
    <nav>
      <ul
        class="flex-row"
      >
        <li
          class="mx-1"
        >
          <a
            href="/signup"
          >
            Signup
          </a>
        </li>
        <li
          class="mx-1"
        >
          <a
            href="/login"
          >
            Login
          </a>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
